app/tests/test_collections_endpoint.py

```diff
@@ -51,6 +52,7 @@ def test_single_collection_endpoint(self):
             msg="Returned data does not match expected data"
         )
 
+    @skip("skipping due to bug https://jira.swisstopo.ch/browse/BGDIINF_SB-1404")
```

Comment:
I would add a comment to the ticket to not forgot about this `skip`

Reply:
done